ql.mm

```diff
@@ -9875,7 +9375,10 @@ Kalmbach axioms (soundness proofs) that require WOML
 $)
 
   $( New equation that holds in Hilbert space, discovered by Pavicic and Megill
-     (unpublished). $)
+     (unpublished).
+
+     [Editor's note:  The date this was added is unknown within the range
+     1997 to 2012.] (Contributed by NM, 1-Jan-1998.) $)
```

Comment:
```suggestion
     [Editor's note:  The date this was added is unknown within the range
     1997 to 29-Oct-2005. The axiom was discovered before 9-Aug-2003.]
     (Contributed by NM, 1-Jan-1998.) $)
```

Reply:
I'm sure that it was in the library by the time the note was added to the main page. I would just update the upper limit to 2003.